charts/tractusx-connector-azure-vault/README.md

```diff
@@ -262,10 +262,10 @@ helm install my-release tractusx-edc/tractusx-connector-azure-vault --version 0.
 | tests | object | `{"hookDeletePolicy":"before-hook-creation,hook-succeeded"}` | Configurations for Helm tests |
 | tests.hookDeletePolicy | string | `"before-hook-creation,hook-succeeded"` | Configure the hook-delete-policy for Helm tests |
 | vault.azure.certificate | string | `nil` |  |
-| vault.azure.client | string | `""` |  |
+| vault.azure.client | string | `"53ba6f2b-6d52-4f5c-8ae0-7adc20808854"` |  |
```

Comment:
since this default value is arbitrary, we should use something like `<AZURE_CLIENT_ID>` instead of a UUID, so that it is very obvious. 

Same is true for `vault.azure.tenant`